jobs/export_process/worker.go

```diff
@@ -21,6 +21,8 @@ type AppIface interface {
 	WriteFile(fr io.Reader, path string) (int64, *model.AppError)
 	BulkExport(ctx request.CTX, writer io.Writer, outPath string, opts model.BulkExportOpts) *model.AppError
 	Log() *mlog.Logger
+	GetID() string
+	SetID(id string)
```

Comment:
I have found alternatives to this. Let's discuss on a call.

Reply:
Ok..